ksqldb-functional-tests/src/test/resources/query-validation-tests/select.json

```diff
@@ -1,8 +1,34 @@
 {
   "comments": [
-    "Tests covering general SELECT clause, a.k.a projection, handling"
+    "Tests covering general SELECT clause, a.k.a projection, handling",
+    "Manually tests `new query planner test` with all key combination:",
+    " - `name` only (as below)",
+    " - `id1` only",
+    " - `id2` only",
+    " - `id2, id1` (reverse order)"
   ],
   "tests": [
+    {
+      "name": "new query planner test",
```

Comment:
Besides these new test, all tests in `select.json` pass now!

I am surprise that the test with `AS` work... Need to do more digging why -- would not have expected them to work without change in the new logical/physical planner?